### PR TITLE
Instructions for Windows download of kubectl.

### PR DESCRIPTION
**Pull request by kw217**
The binary name is kubectl.exe in this case, not kubectl, which is not immediately obvious.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1500)

### CODE REVIEW

**Comment by kw217** [comment]
I've signed it!

**Comment by janetkuo** [comment]
Related #1458

**Comment by kw217** [comment]
I'm working through the linuxfoundation CLA process right now.

**Comment by techtonik** [comment]
The only problem is that Windows doesn't ship with `curl`.

**Comment by kw217** [comment]
@techtonik is that a problem? It's easy to install, and most people trying out minikube will either have it already or know how to get it. Alternatively, we could tell people to download the kubectl.exe using their favourite browser and save it to a relevant location.

**Comment by kw217** [comment]
I've now signed the linuxfoundation CLA.

**Comment by techtonik** [comment]
Well, I use `wget` when I work on console, because it doesn't require fancy output options. And on Windows it is more logical to use `Save link as..` while following tutorial in the browser.

File: docs/getting-started-guides/minikube.md [hunk 1]
```diff
@@ -80,6 +80,8 @@ curl -Lo kubectl http://storage.googleapis.com/kubernetes-release/release/{{page
 curl -Lo kubectl http://storage.googleapis.com/kubernetes-release/release/{{page.version}}.0/bin/darwin/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
 # OS X/386 
 curl -Lo kubectl http://storage.googleapis.com/kubernetes-release/release/{{page.version}}.0/bin/darwin/386/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+# Windows
+curl -Lo kubectl.exe http://storage.googleapis.com/kubernetes-release/release/v1.3.0/bin/windows/amd64/kubectl.exe
```
Comment (techtonik) [inline docs/getting-started-guides/minikube.md]: `{{page.version}}` would be better than hardcoded value. See https://github.com/kubernetes/kubernetes.github.io#use-a-global-variable

**Comment by kw217** [comment]
The link is in a code block, and isn't clickable. Adding a separate link below the block just for Windows users seems excessive. Perhaps I should add "(using curl; or use your browser)" to the comment?

Oops, thanks for spotting the hardcoded value, I forgot that.

**Comment by techtonik** [comment]
I would just leave clickable link for Windows users without `curl`. The only reason why Linux uses command line instead of ordinary download link is that you also need to `chmod` and `sudo` to move the binary into global PATH location.

**Comment by kw217** [comment]
Fair enough!

**Comment by techtonik** [comment]
LGTM. =)

**Comment by techtonik** [comment]
Ouch. There are also binaries to 32-bit Windows.

**Comment by kw217** [comment]
minikube [only has Windows binaries for amd64](https://github.com/kubernetes/minikube/releases) so no need to support x86 kubectl.

**Comment by techtonik** [comment]
Looks like it doesn't support x86 on all platforms, so following this logic, the `curl` links for 386 binaries should be removed for all platforms. But `kubectl` is a client that is not dependent on server architecture.

**Comment by devin-donnelly** [comment]
@kw217, @techtonik, can we resolve this?

**Comment by techtonik** [comment]
@devin-donnelly feel free to merge - I will expand it later.

**Comment by kw217** [comment]
Please can you merge as @techtonik suggests. Thanks.

**Pull request merged**